saturn-datepicker/src/datepicker/datepicker.ts

```diff
@@ -42,22 +42,24 @@ import {
   mixinColor,
   ThemePalette,
 } from '@angular/material/core';
-import {MatDialog, MatDialogRef} from '@angular/material/dialog';
-import {merge, Subject, Subscription} from 'rxjs';
-import {filter, take} from 'rxjs/operators';
-import {SatCalendar} from './calendar';
-import {matDatepickerAnimations} from './datepicker-animations';
-import {createMissingDateImplError} from './datepicker-errors';
-import {SatCalendarCellCssClasses} from './calendar-body';
-import {SatDatepickerInput, SatDatepickerRangeValue} from './datepicker-input';
-import {DateAdapter} from '../datetime/date-adapter';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { merge, Subject, Subscription } from 'rxjs';
+import { filter, take } from 'rxjs/operators';
+import { SatCalendar } from './calendar';
+import { matDatepickerAnimations } from './datepicker-animations';
+import { createMissingDateImplError } from './datepicker-errors';
+import { SatCalendarCellCssClasses } from './calendar-body';
+import { SatDatepickerInput, SatDatepickerRangeValue } from './datepicker-input';
+import { DateAdapter } from '../datetime/date-adapter';
+
+const moment = require('moment');
 
```

Comment:
Nope! No any dependences from moment.js!

Reply:
i did not added moment.js as a new dependency, it was just there. i believe it comes from material datepicker itself.

Reply:
No, it is not. I have a moment in repo (don't remember why), but moment is not required for datepicker. And I think it shouldn't

Reply:
ok i'll update.